src/components/structures/RoomSubList.js

```diff
@@ -119,7 +136,7 @@ var RoomSubList = React.createClass({
     // The header is collapsable if it is hidden or not stuck
     // The dataset elements are added in the RoomList _initAndPositionStickyHeaders method
     isCollapsableOnClick: function() {
-        var stuck = this.refs.header.dataset.stuck;
+        var stuck = this.refs.header.refs.header.dataset.stuck;
```

Comment:
i agree it's ugly, but i don't think there's an alternative having moved the header into the sublist? i don't think there's NPE risk.